apps/vr-tests-react-components/src/stories/InfoButton.stories.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import Screener from 'screener-storybook/src/screener';
+import { InfoButton } from '@fluentui/react-infobutton';
+import { storiesOf } from '@storybook/react';
+import { PopoverProps } from '@fluentui/react-popover';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('InfoButton', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <Screener
+      steps={new Screener.Steps()
+        .snapshot('rest', { cropTo: '.testWrapper' })
+        .hover('#info-button')
+        .snapshot('hover', { cropTo: '.testWrapper' })
+        .focus('#info-button')
+        .snapshot('focus', { cropTo: '.testWrapper' })
+        .mouseDown('#info-button')
+        .snapshot('active', { cropTo: '.testWrapper' })
+        .end()}
+    >
+      {story()}
+    </Screener>
+  ))
+  .addStory(
+    'size',
+    () => (
+      <div style={{ padding: '60px' }}>
+        <InfoButton id="info-button" content="This is the content of an InfoButton." />
+      </div>
+    ),
+    {
+      includeHighContrast: true,
+      includeDarkMode: true,
+    },
+  );
+
+storiesOf('InfoButton - sizes', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <Screener steps={new Screener.Steps().snapshot('active', { cropTo: '.testWrapper' }).end()}>{story()}</Screener>
+  ))
+  .addStory(
+    'size',
+    () => (
+      <div style={{ display: 'flex', flexDirection: 'column', padding: '60px', gap: '80px', alignItems: 'start' }}>
+        <InfoButton size="small" content="This is the content of an InfoButton." />
+        <InfoButton size="medium" content="This is the content of an InfoButton." />
+        <InfoButton size="large" content="This is the content of an InfoButton." />
+      </div>
+    ),
+    {
+      includeHighContrast: true,
+      includeDarkMode: true,
+    },
+  )
+  .addStory(
+    'size-open',
+    () => (
+      <div style={{ display: 'flex', flexDirection: 'column', padding: '60px', gap: '80px', alignItems: 'start' }}>
```

Comment:
It looks like this padding is causing the balloon to go off the edge of the TestWrapper. Maybe change it only to top padding?

![image](https://user-images.githubusercontent.com/48106640/202535078-34f9e747-1fa4-4a64-ad02-b4b24d0d8f71.png)

```suggestion
      <div style={{ display: 'flex', flexDirection: 'column', paddingTop: '60px', gap: '80px', alignItems: 'start' }}>
```